src/reducer/videoReducer.js

```diff
@@ -0,0 +1,9 @@
+const initialVideoState = { videos: [] };
+const videoReducer = (state, { type, payload }) => {
+  switch (type) {
+    case 'INITIAL':
+      return { videos: [...payload] };
+  }
```

Comment:
You should have a default case to avoid unwanted bugs

Reply:
Forget to add one.
Thanks for pointing it out.